apps/i18n/common/en_us.json

```diff
@@ -771,7 +771,7 @@
   "encrypted": "encrypted",
   "end": "end",
   "englishOnly": "English-only",
-  "englishOnlyWarning": "Sorry! This stage is not available in your language. The puzzles in this stage use a mix of English words and characters that can’t be translated right now. You can move on to Stage {nextStage}.",
+  "englishOnlyWarning": "Sorry! This lesson is not available in your language. The levels in this lesson use a mix of English words and characters that can’t be translated right now. You can move on to Lesson {nextStage}.",
```

Comment:
it could be worth finding out if `nextStage` should be updated too.

Reply:
I was gonna tackle updating params used in translations as a later step

Reply:
@daynew does this sound like a safe approach to you?

Reply:
I was also considering not updating the params since I can't update them in all the translations so I think that might break things. Anyway was avoiding for now

Reply:
I think it might be more trouble than it's worth to update those params. As Dani pointed out, the translations would still have the old param name and would look bad at best and break at worst.

Reply:
ok. it may be slightly confusing that the param {stageName} does not match the text, but it should be easy enough for translators to figure it out from its position in the string. I also wanted to avoid changing the string twice in case that caused it to get retranslated twice. So, LGTM with the understanding we are not planning to change the {stageName} param in the future.